### PR TITLE
fix: add option for local_run to emit timestamps

### PR DESCRIPTION
**Pull request by bloodearnest**
This is useful in CI settings.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-VENV_DIR ?= .venv
```
Comment (evansd) [inline Makefile:1]: What's with the Makefile changes?

File: jobrunner/local_run.py [hunk 2]
```diff
@@ -77,6 +77,11 @@ def add_arguments(parser):
         help="Don't stop on first failed action",
         action="store_true",
     )
+    parser.add_argument(
+        "--timestamps",
+        help="Emit timestamps on log out. Useful in CI",
```
Comment (evansd) [inline jobrunner/local_run.py:82]: Nitpick, but the primary audience for the help text here are researchers looking at `opensafely run --help`. So I think the CI comment is just extra noise and potential confusion here. "Log" is also slightly confusing here as there are separate log file which always include timestamps. So I'd just go for:
```
Include timestamps in output
```
The CI thing could go in a Python comment above. And actually, would be worth doing the same for `--continue-on-error`.

**Review by evansd — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (bloodearnest) [reply Makefile:1]: It was broken, so I fixed it so I could use it to run the tests

File: jobrunner/local_run.py (hunk 2 above)
Comment (bloodearnest) [reply jobrunner/local_run.py:82]: Good call. There's already such a comment on the `--continue-on-error` definition :)

File: Makefile (hunk 1 above)
Comment (evansd) [reply Makefile]: Ah, I see. Feels like it belongs a separate commit in that case.

Comment (bloodearnest) [reply Makefile]: reverted

Comment (evansd) [reply Makefile]: :+1: didn't necessarily mean a separate PR btw. Drive by fixes are great, but in my experience it pays off over time to keep commits logically atomic (as far as reasonably possible)

**Pull request merged**